_monitoring-plugins/alerting/api.md

```diff
@@ -1242,6 +1300,8 @@ POST _plugins/_alerting/destinations
 Introduced 1.0
 {: .label .label-purple }
 
+When you update a destination, you can use the `seq_no` and `primary_term` to update an existing destination. If these numbers don’t match the existing destination or the destination doesn’t exist, alerting throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response).
```

Comment:
"When updating a destination, you can include seq_no and primary_term as part of your request. If these numbers don't match the existing monitor or the monitor doesn't exist, the alerting plugin throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response)."

Reply:
Thanks for the review, I fixed the description as you mentioned.